inject/maven-plugin/src/main/java/io/helidon/inject/maven/plugin/TrafficCop.java

```diff
@@ -1,65 +0,0 @@
-/*
- * Copyright (c) 2023 Oracle and/or its affiliates.
```

Comment:
someone on a mac needs to validate they can do a parallel build!

Reply:
It failed on Linux as well. There should be no static loading anymore. I can validate myself (and will validate)

Reply:
Working on a fix (config bean registry was a singleton, making sure it has an instance per service registry)